src/isomorphic/React.js

```diff
@@ -106,7 +110,19 @@ if (__DEV__) {
             'drop-in replacement.',
         );
         warnedForCreateClass = true;
-        return undefined;
+        return createReactClass;
+      },
+    });
+
+    Object.defineProperty(React, 'PropTypes', {
+      get() {
+        warning(
+          didWarnPropTypesDeprecated,
```

Comment:
Let's add something like "it will be removed completely in React 16"

Also, maybe we could make this warning and the one for `React.createClass` more consistent.